src/components/MangaControl.tsx

```diff
@@ -6,11 +6,30 @@ import { TitleIcon, BookmarkIcon, BookOpenIcon, CheckMarkIcon, ColorSwatchIcon,
 
 type Props = {
   readerSettings: ReaderSettings,
-  updateReaderSettings: (newReaderSettings: ReaderSettings) => void
+  updateReaderSettings: (newReaderSettings: ReaderSettings) => void,
+  chapters: string[],
+  chapter: string,
+  updateChapter: (newChapter: string) => void,
 }
 
 const MangaControl = (props: Props) => {
   const [menu, setMenu] = useState(true)
+  const { readerSettings, updateReaderSettings, chapter, chapters, updateChapter } = props;
+
+  const changeReaderMode = (event: React.ChangeEvent<HTMLSelectElement>) => {

```

Comment:
these event handlers are short enough, so you could probably just call them directly in the jsx as an anon function, doesn't matter too much though.